pkg/names/names.go

```diff
@@ -119,10 +119,13 @@ var (
 		{"Nat", "NAT", "nat", regexp.MustCompile("Nat(?!i)", regexp.None)},
 		{"Oidc", "OIDC", "oidc", nil},
 		{"Ocsp", "OCSP", "ocsp", nil},
+		// Model fields starting with 'Ram' refer to RAM
+		{"Ram", "RAM", "ram", regexp.MustCompile("Ram", regexp.None)},
```

Comment:
I *think* this will be OK... I'm not sure if there are fields like `pctRam` or `percentRam` or something like that which we would need to keep an eye out for though.